OpenRA.Mods.Common/Traits/World/SubterraneanLocomotor.cs

```diff
@@ -10,10 +10,12 @@
 #endregion
 
 using System.Collections.Generic;
+using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Traits
 {
 	[Desc("Used by Mobile. Required for subterranean actors. Attach these to the world actor. You can have multiple variants by adding @suffixes.")]
+	[TraitLocation(SystemActors.World | SystemActors.EditorWorld)]
```

Comment:
```suggestion
	[TraitLocation(SystemActors.World)]
```

This and `Locomotor` should actually not be required for mapping afaik.

Reply:
It may in the future for terrain passability information. We already have a usecase for this on aircraft (see recent discussion about aircraft being placed at cruise altitude vs landed).

Reply:
Skipping to be consistent with other Locomotors.